### DISCUSSION TITLE
PBBM, IVIVC, VBE In Person Workshop @ACoP (15 November 2024)

### DISCUSSION THREAD

**Discussion by nicospinu** [Open-Systems-Pharmacology/Forum#1798] [Announcement]
Dear OSP Community,

We would like to invite you to our second workshop, which will officially take place as part of the program at the ACoP Conference in Arizona.

This workshop aims to present the principles/applications of PBBM with hands-on examples using the OSP Suite, PK-Sim® and MoBi®. The learners will:
- Delve into the world of **mechanistic oral absorption modeling** with PK-Sim®, unlocking a deeper understanding of absorption mechanisms.
- Learn how experts are using the OSP suite to forecast **dissolution behavior** and optimize drug formulations.
- Learn about the **in vitro-in vivo correlation** (**IVIVC**) concept and its significance in pharmaceutical development and regulatory applications, from streamlining formulation optimization to expediting regulatory approval processes.
- Explore ways to establish the IVIVC through the OSP Suite.
- Learn about **virtual bioequivalence (VBE)** and dissolution safe space and how these strategies are reshaping traditional approaches in the drug development space.
- Immerse yourself in interactive exercises, gaining experience in VBE analyses with the OSP suite, empowering you to navigate the complexities of pharmaceutical research.

Find more details [here](https://esqlabs.com/event-20240704/). 

Please don't hesitate to reach out to us for any further queries! [training@esqlabs.com](mailto:training@esqlabs.com)

We’re looking forward to meeting you at the ACoP 2024 Conference!

Best wishes,
Nicoleta Spinu
Scientist & Platform Lead Training Business @ESQlabs

**Comment by StephanSchaller** [comment]
PBPK-QSP pre-ACoP Workshop almost sold out!
